config/crd/bases/self-node-remediation.medik8s.io_selfnoderemediationconfigs.yaml

```diff
@@ -98,10 +98,12 @@ spec:
                 default: 180
                 description: SafeTimeToAssumeNodeRebootedSeconds is the time after
                   which the healthy self node remediation agents will assume the unhealthy
-                  node has been rebooted and it is safe to remove the node from the
-                  cluster. This is extremely important. Deleting a node while the
-                  workload is still running there might lead to data corruption and
-                  violation of run-once semantic.
+                  node has been rebooted, and it is safe to remediate the node. This
+                  is extremely important. Remediating a node while the workload is
+                  still running there might lead to data corruption and violation
+                  of run-once semantic. Note that remediation time is affected by
+                  other factors as well, therefore modifying this field will not necessarily
```

Comment:
This is true and more accurate, but it's a terrible message for users.
"Set this, it might do something, it might not"

The CRD is not a substitute for exhaustive documentation, but the fact we can't concisely explain what this field does is surely a sign something needs to be improved.

Reply:
>The CRD is not a substitute for exhaustive documentation

Agreed, I assume that when/after we'll merge this CR will consider best approach for ocp docs

>This is true and more accurate, but it's a terrible message for users.
"Set this, it might do something, it might not"

How about something like this ? ``Note that SafeTimeToAssumeNodeRebootedSeconds field is not meant for modifying overall remediation time ,but to verify remediation takes place once fencing is completed - therefore modifying this field will not necessarily affect the overall remediation time.``

Reply:
That's not really true though.  It *is* meant to control the overall remediation time, but not below a level calculated from other variables.

Reply:
In what way would they be different goals?

There are two times:
a) the maximum amount of time between a failure and node committing sepuku
b) the amount of time surviving peers wait before assuming the node is in a safe state (ie. dead)

They are related because while 'b' can be as long as the user wants, it must never be smaller than 'a'.

Reply:
I've modified the comment per our conversion, feel free to edit more